src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -73,10 +66,11 @@ public Events(FlareBot bot) {
 
     @Override
     public void onMessageReactionAdd(MessageReactionAddEvent event) {
-        if(!event.getGuild().getSelfMember().hasPermission(event.getTextChannel(), Permission.MESSAGE_READ)) return;
+        if (!event.getGuild().getSelfMember().hasPermission(event.getTextChannel(), Permission.MESSAGE_READ)) return;
         if (!event.getGuild().getId().equals(FlareBot.OFFICIAL_GUILD)) return;
         event.getChannel().getMessageById(event.getMessageId()).queue(message -> {
-            MessageReaction reaction = message.getReactions().stream().filter(r -> r.getEmote().getName().equals(event.getReactionEmote().getName())).findFirst().orElse(null);
+            MessageReaction reaction =
+                    message.getReactions().stream().filter(r -> r.getEmote().getName().equals(event.getReactionEmote().getName())).findFirst().orElse(null);
```

Comment:
This formatting thing triggers me so hard :D

Reply:
It's on a new line so what? xD

Reply:
Triggers me is all, literally an = then an indented assignment. *sigh* so dumb